src/common/utxobased/engine/makeUtxoEngine.ts

```diff
@@ -539,6 +539,34 @@ export async function makeUtxoEngine(
       })
       await tmpState.start()
       return end
+    },
+
+    otherMethods: {
+      signMessageBase64: async (
+        message: string,
+        address: string
+      ): Promise<string> => {
+        const scriptPubkey = walletTools.addressToScriptPubkey(address)
+        const processorAddress = await processor.fetchAddress(scriptPubkey)
+        if (processorAddress?.path == null) {
+          throw new Error('Missing address to sign with')
+        }
+
+        // Derive the xprivs on the fly, since we do not persist them
+        const xprivKeys = await fetchOrDeriveXprivFromKeys({
```

Comment:
Need to resolve conflict with new keys PR.

Reply:
Done.